merlin/models/tf/models/base.py

```diff
@@ -13,8 +15,13 @@
 from merlin.models.utils.dataset import unique_rows_by_features
 from merlin.schema import Schema, Tags
 
+if TYPE_CHECKING:
+    from merlin.models.tf.blocks.core.index import TopKIndexBlock
+
 
 class MetricsComputeCallback(tf.keras.callbacks.Callback):
+    """Callback that handles when to compute metrics."""
+
     def __init__(self, train_metrics_steps=1, **kwargs):
         self.train_metrics_steps = train_metrics_steps
         self._is_fitting = False
```

Comment:
I believe after removing `model._should_compute_eval_metrics_as_in_training`, the `self._is_fitting` is now useless